api/config/application.rb

```diff
@@ -24,6 +24,13 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
 
+    config.before_configuration do
```

Comment:
we add the environment variables here

Reply:
We should actually do this in `development.rb` so it only runs in dev mode (locally).